examples/exchange_api_examples/accounts_rpc/7_OrderStatesRequest.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Injective Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Injective Exchange API client for Python. Example only."""
+
+import asyncio
+import logging
+
+from pyinjective.client import Client
+from pyinjective.constant import Network
+
+async def main() -> None:
```

Comment:
small thing, I have seen a lot of async used for main() here, it is unnecessary imo. And the function is not async here anyway